classes/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Commands for the Action Scheduler by Prospress.
+ */
+class ActionScheduler_WPCLI_Scheduler_command extends WP_CLI_Command {
+
+	/**
+	 * Run the Action Scheduler
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--batch-size=<size>]
+	 * : The maximum number of actions to run. Defaults to 100.
+	 *
+	 * [--force]
+	 * : Whether to force execution despite the maximum number of concurrent processes being exceeded.
+	 */
+	public function run( $args, $assoc_args ) {
+		// Handle passed arguments.
+		$batch = \WP_CLI\Utils\get_flag_value( $assoc_args, 'batch-size', 100 );
+		$force = \WP_CLI\Utils\get_flag_value( $assoc_args, 'force', false );
+
+		$completed = 0;
+
+		try {
+			// Set up the class instances we'll need
+			$store   = ActionScheduler_Store::instance();
+			$monitor = new ActionScheduler_FatalErrorMonitor( $store );
+			$cleaner = new ActionScheduler_QueueCleaner( $store );
```

Comment:
The main reason for setting up the constructor of `ActionScheduler_WPCLI_QueueRunner` without defaults was to be a bit more OOP where possible. But I guess in this case it would make sense to stick with what is already there since the parent class will now handle things for us.